metrics_utility/management/commands/build_report.py

```diff
@@ -61,23 +67,18 @@ def init_logging(self):
         handler.setLevel(logging.DEBUG)
         handler.setFormatter(logging.Formatter('%(message)s'))
         self.logger.addHandler(handler)
-        self.logger.propagate = False
+        self.logger.propagate = True
 
-    def handle(self, *args, **options):
+    def _handle(self, *args, **options):
         self.init_logging()
+        opt_month, month, next_month, opt_since, opt_until, opt_ephemeral = validate_date_extra_params(options)
 
-        # parse params
-        opt_month = options.get('month') or None
-        opt_month, month, next_month = self._handle_month(opt_month)
-
-        # Since and ephemenral params are specific to subset of reports
-        opt_since = None
-        opt_ephemeral = None
+        if opt_month is None:
```

Comment:
surprising .. if validate_* handles setting opt_month, why doesn't it handle setting opt_since and opt_until?